packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -250,7 +289,17 @@ function WorklistDataGrid(props) {
   useEffect(() => {
     setDataGridColumns(displayedColumns.map((column) => initializeColumn(column)));
   // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [columns]);
+  }, [pinnedColumns, overflowColumns]);
+
+  // useEffect for pinned column offset
+  useEffect(() => {
+    if (pinnedColumns.length > 0) {
+      calculateOffsets();
+    }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [dataGridColumns]);
```

Comment:
You would want to leverage the useEffect that if dependent upon the columns prop.  You don't need an addition useEffect hook.

Reply:
We're using a useEffect that is dependent on the dataGridColumns variable to account for the change in column widths after resizing. When this is done in the useEffect that is dependent on the columns props, the pinned column offsets won't update when columns are resized.